cardano-db-sync/src/Cardano/DbSync/Types.hs

```diff
@@ -29,3 +45,21 @@ newtype GenesisFile = GenesisFile
 newtype SocketPath = SocketPath
   { unSocketPath :: FilePath
   }
+
+
+type ShelleyBlock = Shelley.ShelleyBlock Shelley.TPraosStandardCrypto
```

Comment:
Ugh, types. I guess this is fixing them to be monomorphic. Not sure if this will work in the future.

Reply:
They are used mono-morphically throughout the `node` repo.